neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -432,6 +438,18 @@ def _get_init_experiment_operations(self, name, entrypoint, params, tags) -> Lis
             path=alpha_path_utils.parse_path(alpha_consts.SYSTEM_NAME_ATTRIBUTE_PATH),
             value=name,
         ))
+        # Assign experiment description
+        description = description or ""
```

Comment:
Most of the checks if there's some non empty values passed  are checked on `Project` level [here](https://github.com/neptune-ai/neptune-client/blob/230f8231870302aa85d102c992755a95172e4482/neptune/projects.py#L385).

At this level we should be sure that all passed arguments are given by user or replaced with default values.